les/handler.go

```diff
@@ -87,12 +87,16 @@ type txPool interface {
 }
 
 type ProtocolManager struct {
-	lightSync    bool
+	// Configs
+	chainConfig *params.ChainConfig
+	iConfig     *light.IndexerConfig
+
+	light     bool   // The indicator whether the node is light client
```

Comment:
Let's call this `client` instead. Since we're in `les`, both server and client are `light`.